From 58222cee083f1c09c3a90ec72c18b4db686e04f2 Mon Sep 17 00:00:00 2001
From: Simon Josefsson <simon@josefsson.org>
Date: Fri, 19 Jan 2024 09:57:07 +0100
Subject: [PATCH] Disable test that require sourceless test/data/helloworld
 binary.
Forwarded: https://github.com/in-toto/in-toto-golang/issues/297

---
 in_toto/runlib_test.go | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/in_toto/runlib_test.go b/in_toto/runlib_test.go
index 0f00dac..b272a79 100644
--- a/in_toto/runlib_test.go
+++ b/in_toto/runlib_test.go
@@ -655,17 +655,6 @@ func TestRecordArtifactWithBlobs(t *testing.T) {
 				"sha512": "bb040966a5a6aefb646909f636f7f99c9e16b684a1f0e83a87dc30c3ab4d9dec2f9b0091d8be74bbc78ba29cb0c2dd027c223579028cf9822b0bccc49d493a6d"},
 			wantErr: nil,
 		},
-		{
-			name: "test binary blob with windows-like line breaks as byte segments",
-			args: args{
-				path:           "helloworld",
-				hashAlgorithms: []string{"sha256", "sha384", "sha512"},
-			},
-			want: map[string]interface{}{"sha256": "fd895747460401ca62d81f310538110734ff5401f8ef86c3ab27168598225db8",
-				"sha384": "ddc3ac40ca8d04929e13c42d555a5a6774d35bfac9e2f4cde5847ab3f12f36831faa3baf1b33922b53d288b352ae4b9a",
-				"sha512": "46f0e37e72879843f95ddecc4d511c9ba90241c34b471c2f2caca2784abe185da50ddc5252562b2a911b7cfedafa3e878f0e6b7aa843c136915da5306061e501"},
-			wantErr: nil,
-		},
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-- 
2.34.1

